spec/redirect_report_spec.rb

```diff
@@ -130,6 +137,44 @@ def self.foreach(name)
 EOD
       end
     end
+
+    context 'mailing the report' do
+      include Mail::Matchers
+
+      let :log_line do
+        '5.148.140.228 - - [16/Feb/2015:15:39:36 +0000] "GET /article/ HTTP/1.1" 302 178 "-" "-" 0.000 0.000 [-] [-]'
+      end
+
+      before do
+        RedirectReport.run(['production.log', '-m', 'misha.gorodnitzky@moneyadviceservice.org.uk'])
+      end
+
+      it do
+        should have_sent_email.to('misha.gorodnitzky@moneyadviceservice.org.uk')
+      end
+
+      it do
+        should have_sent_email.from 'development.team@moneyadviceservice.org.uk'
+      end
+
+      it do
+        should have_sent_email.with_subject 'Redirects Report'
+      end
```

Comment:
RSpec specifically supports ["one-liner" syntax](https://www.relishapp.com/rspec/rspec-core/v/3-0/docs/subject/one-liner-syntax) for situations like this where you can express better what you want in code (vs using descriptive text).